tests/e2e/testsLibrary/LsTests.ts

```diff
@@ -8,81 +8,88 @@
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
 
-import { e2eContainer } from '../inversify.config';
-import { Editor, CLASSES, Ide, TimeoutConstants } from '..';
+import 'reflect-metadata';
+import { inject, injectable } from 'inversify';
+import { CLASSES } from '../inversify.types';
+import { TimeoutConstants } from '../TimeoutConstants';
+import { Editor } from '../pageobjects/ide/Editor';
+import { Ide} from '../pageobjects/ide/Ide';
 import { Key, error } from 'selenium-webdriver';
 import { Logger } from '../utils/Logger';
 
-const editor: Editor = e2eContainer.get(CLASSES.Editor);
-const ide: Ide = e2eContainer.get(CLASSES.Ide);
+@injectable()
+export class LsTests {
 
- export function errorHighlighting(openedTab: string, textToWrite: string, line: number) {
-    test('Error highlighting', async () => {
-        await editor.type(openedTab, textToWrite, line);
-        try {
-            await editor.waitErrorInLine(line);
-        } catch (err) {
-            if (!(err instanceof error.TimeoutError)) {
-                throw err;
+    constructor(@inject(CLASSES.Editor) private readonly editor: Editor, @inject(CLASSES.Ide) private readonly ide: Ide) { }
+
+    public errorHighlighting(openedTab: string, textToWrite: string, line: number) {
+        test('Error highlighting', async () => {
+            await this.editor.type(openedTab, textToWrite, line);
+            try {
+                await this.editor.waitErrorInLine(line);
+            } catch (err) {
+                if (!(err instanceof error.TimeoutError)) {
+                    throw err;
+                }
+                // try again
+                Logger.debug(`Waiting for squiggly-error failed for the first time (issue #16166)`);
```

Comment:
Looks like this issue is closed
https://github.com/eclipse/che/issues/16166
What about remove this try/catch ?

Reply:
thanks for noticing, I'll verify the issue and fix the try-catch block